<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1076</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Avalon Pilot GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,41,59,255)</string>
   </property>
   <widget class="QLabel" name="MainDisplay">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>900</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MainDisplay_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>900</width>
      <height>475</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Front">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Left">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Right">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Right_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>860</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Left_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>770</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Front_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QLabel" name="telemetry_label">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>100</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>270</y>
      <width>165</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Pitch PID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>270</y>
      <width>176</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Depth PID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>100</y>
      <width>311</width>
      <height>127</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="1">
      <widget class="QComboBox" name="COMport_list">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>ROV status</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="serial_state_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Not Connected</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Serial COM port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Joystick status</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="serial_state_label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Not Connected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>510</y>
      <width>378</width>
      <height>226</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_24">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Vertical Front</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Back Left</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Manual Thrusters Control</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Front Left</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Front Right</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Back Right</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Vertical Back</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_19"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_18"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_17"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_16"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>360</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0,255,255,100);color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color: rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Calibrate Depth</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: 
                                 rgba(0,255,255,100);
                                 color: rgba(0,190,255,255);
                                 border-style: solid;
                                 border-radius: 3px;
                                 border-width: 0.5px;
                                 border-color: 
                                 rgba(0,140,255,255);</string>
       </property>
       <property name="text">
        <string>Calibrate IMU</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="telemetry_label_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>290</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0,190,255,255);border-style: solid;border-radius: 3px;border-width: 0.5px;border-color:rgba(0,140,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
